src/Magnum/GL/Buffer.cpp

```diff
@@ -468,11 +468,17 @@ void Buffer::getParameterImplementationDSA(const GLenum value, GLint* const data
 }
 #endif
 
-#ifndef MAGNUM_TARGET_GLES
+#if !defined(MAGNUM_TARGET_GLES) || (defined(MAGNUM_TARGET_WEBGL) && !defined(MAGNUM_TARGET_GLES2))
 void Buffer::getSubDataImplementationDefault(const GLintptr offset, const GLsizeiptr size, GLvoid* const data) {
+    #if !defined(CORRADE_TARGET_EMPSCRIPTEN) || (__EMSCRIPTEN_major__*10000 + __EMSCRIPTEN_minor__*100 + __EMSCRIPTEN_tiny__ >= 20017)
```

Comment:
~~Heh. I can't see what's wrong with the check here. CI has 1.39.6 and yet it somehow evaluates as true :thinking:~~

Ah, you have `EMPSCRIPTEN` here, haha :)

Reply:
👀 Good catch